src/components/Inbox/index.tsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemText from '@material-ui/core/ListItemText';
+import ListSubheader from '@material-ui/core/ListSubheader';
+import { Checkbox, Divider, IconButton } from '@material-ui/core';
+import { StylesProvider } from '@material-ui/core/styles';
+import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
+
+import './styles.scss';
+
+const Inbox: React.FunctionComponent = () => (
+  <StylesProvider injectFirst>
```

Comment:
j curious what does this do

Reply:
something along the lines of prioritize the custom css rules over the material ui native css rules